_pytest/mark.py

```diff
@@ -259,7 +263,13 @@ def matchkeyword(colitem, keywordexpr):
         return mapping[keywordexpr]
     elif keywordexpr.startswith("not ") and " " not in keywordexpr[4:]:
         return not mapping[keywordexpr[4:]]
-    return eval(keywordexpr, {}, mapping)
+    for kwd in keywordexpr.split():
+        if keyword.iskeyword(kwd) and kwd not in python_keywords_allowed_list:
+            raise AttributeError("Python keyword '{}' not accepted in expressions passed to '-k'".format(kwd))
```

Comment:
I think raising `UsageError` might be more appropriate and provide a better user experience because it won't show up as an internal error.

Reply:
fixed